cli/cmd/errors.go

```diff
@@ -219,3 +230,45 @@ func ErrorInvalidOperatorEndpoint(endpoint string) error {
 		Message: fmt.Sprintf("%s is not a cortex operator endpoint; run `cortex cluster info` to show your operator endpoint or run `cortex cluster up` to spin up a new cluster", endpoint),
 	})
 }
+
+func ErrorClusterDoesNotExist(clusterName string, region string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrClusterDoesNotExist,
+		Message: fmt.Sprintf("cluster %s in %s does not exist", clusterName, region),
+	})
+}
+
+func ErrorClusterUpInProgress(clusterName string, region string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrClusterUpInProgress,
+		Message: fmt.Sprintf("creation of cluster %s in %s is already in progress", clusterName, region),
+	})
+}
+
+func ErrorClusterAlreadyCreated(clusterName string, region string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrClusterAlreadyCreated,
+		Message: fmt.Sprintf("cluster %s in %s has already been created", clusterName, region),
+	})
+}
+
+func ErrorClusterDownInProgress(clusterName string, region string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrClusterDownInProgress,
+		Message: fmt.Sprintf("deletion of cluster %s in %s is already in progress", clusterName, region),
+	})
+}
+
+func ErrorClusterAlreadyDeleted(clusterName string, region string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrClusterAlreadyDeleted,
+		Message: fmt.Sprintf("cluster %s in %s has already been deleted", clusterName, region),
+	})
+}
+
+func ErrorFailedClusterStatus(status clusterstate.Status, clusterName string, region string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrFailedClusterStatus,
+		Message: fmt.Sprintf("cluster %s in %s encountered an unexpected status %s, please try to delete the cluster with `cortex cluster down` or delete the cloudformation stacks manually on your AWS console %s", clusterName, region, string(status), getCloudFormationURL(clusterName, region)),
```

Comment:
"on your AWS console %s" -> "in your AWS console: %s" or "in your AWS console (%s)"